core/BaseComponent.js

```diff
@@ -323,6 +334,11 @@ export class BaseComponent extends HTMLElement {
       for (let proc of this.tplProcessors) {
         this.tplProcessors.delete(proc);
       }
+      styleMutationObserverCbList?.delete(this.updateCssData);
+      if (!styleMutationObserverCbList.size) {
+        styleMutationObserver.disconnect();
```

Comment:
Should we also destroy observer on disconnectCallback?

Reply:
didn't get that